Docs for initial release of Forms 14 (#6118)
umbraco/UmbracoDocs: 14/umbraco-forms/upgrading/version-specific.md

# Version Specific Upgrade Notes

> This page covers specific upgrade documentation for when migrating to major 14 of Umbraco Forms.&#x20;

Reviewer: ```suggestion
This page provides specific upgrade documentation for migrating to Umbraco Forms version 14.
```

{% hint style="info" %}
If you are upgrading to a new minor or patch version, you can find information about the breaking changes in the [Release Notes](../release-notes.md) article.
{% endhint %}

## Version Specific Upgrade Notes History

Version 14 of Umbraco Forms has a minimum dependency on Umbraco CMS core of `14.0.0`. It runs on .NET 8.

#### **Breaking changes**

Version 14 contains a number of breaking changes, primarily due to the new backoffice introduced in Umbraco 14. The details are listed here:

#### **Behavior**

* A new management API has been introduced at `umbraco/forms/management/api`.
* The root of the existing delivery API used for headless/AJAX solutions has moved to `umbraco/forms/delivery/api`.

> * The HTML helper `RenderUmbracoFormDependencies` no longer renders the promises polyfill, which isn't needed in modern browsers.

Reviewer: ```suggestion
* The HTML helper `RenderUmbracoFormDependencies` no longer renders the promises polyfill, which is not needed in modern browsers.
```

* Server-side registration of content apps has been removed as this is now a client-side concern.

> * Creation of custom field, workflow and other provider types remains primarily a server-side task. However they no longer require the provision of AngularJS views and controllers. Instead these reference registered client-side manifests. Read more on this under [extending Umbraco Forms](../developer/extending/README.md).

Reviewer: ```suggestion
* Creation of custom fields, workflow, and other provider types remains primarily a server-side task. However, they no longer require the provision of AngularJS views and controllers. Instead, these reference registered client-side manifests. For more information, see the [extending Umbraco Forms](../developer/extending/README.md) article.
```
